betty/extension/webpack/webpack/webpack.config.js

```diff
@@ -0,0 +1,186 @@
+    const entryNames = document.getElementById('webpack-entry-loader').dataset.webpackEntryLoader.split(':')
+    const entryScripts = new Set(entryNames.reduce(
+        (accumulatedEntryScripts, entryName) => [...accumulatedEntryScripts, ...entryScriptsMap[entryName]],
+        [],
+    ))
+    await Promise.allSettled([...entryScripts].map(async (entryScript) => await import(entryScript)))
+}
+load()
+`
+              compilation.emitAsset(
+                path.join('js', 'webpack-entry-loader.js'),
+                new webpack.sources.RawSource(webpackEntryLoader)
+              )
+              return callback()
+            }
+          )
+        }
+      )
+    })
+  }
+}
+
+const webpackConfiguration = {
+  mode: configuration.debug ? 'development' : 'production',
+  devtool: configuration.debug ? 'eval-source-map' : false,
+  entry: configuration.entry,
+  output: {
+    path: path.resolve(__dirname, 'build'),
+    filename: 'js/[name].js'
+  },
+  optimization: {
+    concatenateModules: true,
+    minimize: !configuration.debug,
+    minimizer: [
+      new CssMinimizerPlugin(),
+      new TerserPlugin({
+        extractComments: false,
+        terserOptions: {
+          output: {
+            comments: false
+          }
+        }
+      })
+    ],
+    splitChunks: {
+      chunks: 'all',
+      cacheGroups: {
+        // The resulting CSS files are one per entrypoint, and a single vendor.css.
+        // This make for easy importing.
+        vendorCss: {
+          test: /[\\/]node_modules[\\/].+?\.css$/,
+          name: 'vendor',
+          priority: -10
+        },
+        vendorJs: {
+          test: /[\\/]node_modules[\\/].+?\.js$/,
+          priority: -10
+        }
+      }
+    },
+    runtimeChunk: 'single'
+  },
+  plugins: [
+    new EntryScriptCollector(),
+    new CleanWebpackPlugin(),
+    new MiniCssExtractPlugin({
+      filename: 'css/[name].css'
+    })
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.(js|ts)$/,
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              cacheDirectory: configuration.cacheDirectory,
+              presets: [
+                [
+                  '@babel/preset-env', {
+                    debug: configuration.debug,
+                    modules: false,
+                    useBuiltIns: 'usage',
+                    corejs: 3
+                  },
+                ],
+                '@babel/preset-typescript',
+              ]
+            }
+          }
+        ]
+      },
+      {
+        test: /\.s?css$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              publicPath: '/'
+            }
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              url: {
+                filter: (url, resourcePath) => resourcePath.includes('/node_modules/'),
```

Comment:
Document that this is because Betty's own assets are generated through the assets API